common/core/web/keyboard-processor/src/text/keyboardProcessor.ts

```diff
@@ -108,8 +108,7 @@ namespace com.keyman.text {
      * @param   {boolean} usingOSK
```

Comment:
```suggestion
     * @param   {OutputTarget} outputTarget   ____________________
```

This suggestion needs some text filled in...